### ISSUE TITLE
[Torrent Downloads] not parsing pubDate

### ISSUE THREAD

**Issue by tcamargo** [Jackett/Jackett#2081]
Hi,

Jackett is not, correctly, parsing to the pubdate for Torrent Downloads. It is easy to replicate using manual search:

```
2017y ago | Torrent Downloads | Injustice 2 030 (2017) (digital) (Son of Ultron-​Empire) cbr -- | -- | --
```

Bellow is the corresponding log for an api call:

```
2017-11-06 19:24:30 | WARNING | Unable to parse posting date from provider result set for :Injustice 2 030 (2017) (digital) (Son of Ultron-Empire) cbr
-- | -- | --
2017-11-06 19:24:30 | WARNING | Unable to convert timestamp from : (None,) ['NoneType' object has no attribute '__getitem__']
2017-11-06 19:24:30 | INFO | no errors on data retrieval...proceeding
```

And the returned XML:
```
<item>
      <title>Injustice 2 030 (2017) (digital) (Son of Ultron-Empire) cbr</title>
      <guid>https://www.torrentdownloads.me/torrent/1663691832/Injustice+2+030+(2017)+(digital)+(Son+of+Ultron-Empire)+cbr</guid>
      <comments>https://www.torrentdownloads.me/torrent/1663691832/Injustice+2+030+(2017)+(digital)+(Son+of+Ultron-Empire)+cbr</comments>
      <pubDate>2017-11-06T19:35:12.68307+00:00</pubDate>
      <size>24620564</size>
      <description />
      <link>http://192.168.2.150:9117/dl/torrentdownloads/?jackett_apikey=6x1xt29ior6v2ds1wz3rut2v2ie4e6ic&amp;path=WGlrTFpNK3IxbzhVNlpaU3ZMN1VmNk9JMVZYaldKMVVBbnY3TE5FQmlPemcvQWF3VFY4MWx2Mjl0SlFaWEhCd3lDamg2TnQ0bkUxT0tUaGVIWTY1bjlGUkxySndCUjFrZ0Q3NmZQdnRVQWh5ZTlKaXFEYk9MRnJpN2tFVTgwT3NJaWtEeXJUTS9yMG5PdnZORm5HaE1EZWN2MVlXY1Uva3kzNTFjb1k3cU5xN3RMZWtDOHdLWnhrYkpnNk83V1QyRUMvZllndGZjb3BXZGF4QjRydXV3bG5DQ0hhdUxqWkVVd0NMNTdreVB4RlR3dXE2b1ZnQkdwTDAvMy9hLzlBTzc1NHk3RnRtMXMxSldISEV2QUhhZE1LUENDSWN6bUJnRjY5SzFmOWZGOVpUZ3dHYmJUT1BIQW9sdEYyRUNRRDBEWHFkV3pQZlVLSTBtM2JqZVFDdjIvMm5PdlNCRkY4TEJZOFVmcHpnRE5SdWZrempOV1pnbU9PdVdQQVN0YzdsNUpUYmFWU044NjRXaTJOOFE0M0F5amlpVHR1ZjlZVUw0QjBsTkFGOXNRZz01&amp;file=Injustice+2+030+(2017)+(digital)+(Son+of+Ultron-Empire)+cbr.torrent</link>
      <category>7000</category>
      <category>100009</category>
      <enclosure url="http://192.168.2.150:9117/dl/torrentdownloads/?jackett_apikey=6x1xt29ior6v2ds1wz3rut2v2ie4e6ic&amp;path=WGlrTFpNK3IxbzhVNlpaU3ZMN1VmNk9JMVZYaldKMVVBbnY3TE5FQmlPemcvQWF3VFY4MWx2Mjl0SlFaWEhCd3lDamg2TnQ0bkUxT0tUaGVIWTY1bjlGUkxySndCUjFrZ0Q3NmZQdnRVQWh5ZTlKaXFEYk9MRnJpN2tFVTgwT3NJaWtEeXJUTS9yMG5PdnZORm5HaE1EZWN2MVlXY1Uva3kzNTFjb1k3cU5xN3RMZWtDOHdLWnhrYkpnNk83V1QyRUMvZllndGZjb3BXZGF4QjRydXV3bG5DQ0hhdUxqWkVVd0NMNTdreVB4RlR3dXE2b1ZnQkdwTDAvMy9hLzlBTzc1NHk3RnRtMXMxSldISEV2QUhhZE1LUENDSWN6bUJnRjY5SzFmOWZGOVpUZ3dHYmJUT1BIQW9sdEYyRUNRRDBEWHFkV3pQZlVLSTBtM2JqZVFDdjIvMm5PdlNCRkY4TEJZOFVmcHpnRE5SdWZrempOV1pnbU9PdVdQQVN0YzdsNUpUYmFWU044NjRXaTJOOFE0M0F5amlpVHR1ZjlZVUw0QjBsTkFGOXNRZz01&amp;file=Injustice+2+030+(2017)+(digital)+(Son+of+Ultron-Empire)+cbr.torrent" length="24620564" type="application/x-bittorrent" />
      <torznab:attr name="category" value="7000" />
      <torznab:attr name="category" value="100009" />
      <torznab:attr name="seeders" value="49" />
      <torznab:attr name="peers" value="52" />
      <torznab:attr name="minimumratio" value="1" />
      <torznab:attr name="minimumseedtime" value="172800" />
      <torznab:attr name="downloadvolumefactor" value="1" />
      <torznab:attr name="uploadvolumefactor" value="1" />
    </item>
```

Jackett version:  Jackett Version 0.8.355.0
Mono version (if not using Windows):  Mono JIT compiler version 5.4.0.201 (tarball Fri Oct 13 21:51:45 UTC 2017)

**Comment by flightlevel** [comment]
The manual search page shows 2017 years ago as their website doesn't display a date when searching (thats the source for Jackett)

For that API log error, what program is that from and does it use torznab or torrentpotato?

**Comment by tcamargo** [comment]
I am using Mylar with torznab.

**Comment by kaso17** [comment]
we were using the wrong format in case the pubDate isn't available.
Fixed via 5a555427eec35a26342eb13983fe439574cc3fb9
